samples/identify-graphics/README.metadata.json

```
@@ -0,0 +1,27 @@
+{
+    "category": "Search and Query",
+    "description": "Display a message when a graphic on the map is tapped.",
+    "formal_name": "IdentifyGraphics",
+    "ignore": false,
+    "images": [
+        "identify-graphics.png"
+    ],
+    "keywords": [
+        "geoview-compose",
+        "identify",
+        "graphic"
```

On applying the suggestions from the metadata checker: 
```suggestion
        "graphics",
        "identify",
        "Graphic",
        "GraphicsOverlay",
        "IdentifyGraphicsOverlayResult",
        "MapView",
        "MapViewProxy"
```